layer/layer.go

```diff
@@ -173,7 +173,7 @@ type Store interface {
 	Map() map[ChainID]Layer
 	Release(Layer) ([]Metadata, error)
 
-	CreateRWLayer(id string, parent ChainID, mountLabel string, initFunc MountInit, storageOpt map[string]string) (RWLayer, error)
+	CreateRWLayer(id string, parent ChainID, mountLabel string, initFunc MountInit, storageOpt map[string]string, sharedRootFs bool) (RWLayer, error)
```

Comment:
May be something like `mockLayerStore` which might not want to implement `ShareableStore` interface?